zitadel/terraform/main.tf

```diff
@@ -0,0 +1,95 @@
+terraform {
+  required_providers {
+    zitadel = {
+      source  = "zitadel/zitadel"
+      version = "1.0.0-alpha.18"
+    }
+  }
+}
+
+provider "zitadel" {
+  domain           = "iam.xpanse.site"
+  insecure         = "false"
+  port             = "443"
+  jwt_profile_file = "local-token.json"
+}
+
+resource "zitadel_org" "eclipse-xpanse" {
+  name = "eclipse-xpanse"
+}
+
+resource zitadel_project eclipse-xpanse {
+  name                     = "eclipse-xpanse"
+  org_id                   = zitadel_org.eclipse-xpanse.id
+  project_role_assertion   = true
+  project_role_check       = false
+  has_project_check        = false
+  private_labeling_setting = "PRIVATE_LABELING_SETTING_ENFORCE_PROJECT_RESOURCE_OWNER_POLICY"
+}
+
+resource zitadel_application_api application_api {
+  org_id           = zitadel_org.eclipse-xpanse.id
+  project_id       = zitadel_project.eclipse-xpanse.id
+  name             = "xpanse-api"
+  auth_method_type = "API_AUTH_METHOD_TYPE_PRIVATE_KEY_JWT"
+}
+
+resource zitadel_application_oidc xpanse-ui {
+  project_id = zitadel_project.eclipse-xpanse.id
+  org_id     = zitadel_org.eclipse-xpanse.id
+
+  name                        = "xpanse-ui"
+  redirect_uris               = ["https://localhost:3000"]
```

Comment:
we must move this value to a variables.tf file

Reply:
fixed